docs/en/plugins/grpc.md

```diff
@@ -1,20 +1,299 @@
-# gRPC plugin
+# Plugins — gRPC
```

Comment:
We might also mention (if you think it worth mentioning in this section) that all of our proto-API can be found here: https://buf.build/roadrunner-server/api